source/_components/saj.markdown

```diff
@@ -0,0 +1,58 @@
+---
+title: "SAJ Solar Inverter"
+description: "Instructions on how to connect your SAJ Solar Inverter to Home Assistant."
+ha_category:
+  - Energy
+logo: saj.png
+ha_iot_class: Local Polling
+ha_release: "0.100"
+redirect_from:
+ - /components/sensor.saj/
+---
+
+The `saj` sensor will poll a [SAJ](https://www.saj-electric.com/) solar inverter and present the values as sensors in Home Assistant.
+
+This sensor uses the web interface and to use it, you have to be able to connect to the solar inverter from your favorite web browser.
+
+## Configuration
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: saj
+    host: IP_ADDRESS_OF_DEVICE
+    monitored_conditions:
```

Comment:
Only required variables in the first example

Reply:
Changed